docsite/rst/galaxy.rst

```diff
@@ -169,7 +181,9 @@ The search command will return a list of the first 1000 results matching your se
 
 .. note::
 
-    The format of results pictured here is new in Ansible 2.0.
+    The search command in Ansible 2.0 requires using the Galaxy 2.0 Beta site. Use the ``--server`` option to access 
+    `https://galaxy-qa.ansible.com <https://galaxy-qa.ansible.com>`_. You can also add a *server* definition in the [galaxy] 
```

Comment:
if the search command cannot be used yet, i would prefer we remove it and add it back when it is available.